### ISSUE TITLE
web infrastructure is not all under W3C's scope

### ISSUE THREAD

**Issue by dontcallmedom** [darobin/w3c-structure#4]
I like the characterization of W3C's mission to be focused on web infrastructure; but the current articulation in the introduction makes it sound like all web infrastructure is under W3C's scope.

This document focuses specifically on revising the governance structure of W3C - which is fine, but it should be made clearer (if only because there may be a useful broader conversation to be had in managing that infrastructure across different governance systems)

**Comment by mnot** [comment]
I agree that the use of 'infrastructure' here is likely to be confusing to many.

**Comment by darobin** [comment]
I've clarified that the W3C is only working on some of that infra, that only part of the infra is under the community's current remit. LMK what you think.

**Comment by dontcallmedom** [comment]
https://github.com/darobin/w3c-structure/commit/3fb83711d1a9f2d1304bc182dfe7e9e030e206e4 LGTM, thanks
